source/includes/_install_android.md

```diff
@@ -5,15 +5,15 @@
 <dependency>
     <groupId>com.wootric</groupId>
     <artifactId>wootric-sdk-android</artifactId>
-    <version>2.6.1</version>
+    <version>2.7.0</version>
```

Comment:
`2.7.1` or `2.7.0`?

Reply:
It's `2.7.0` @diegoserranoa

Reply:
Ok, got mislead by the PR title. If you can fix it :)

Reply:
Oops looks like I did write that heading wrong 😠